### ISSUE TITLE
sankakucomplex - no download speed + multiple errors on saving + task info

### ISSUE THREAD

**Issue by 4Sol** [KurtBestor/Hitomi-Downloader#6943]
Hello
After an update I have no speed downloading from the site
It counts all the images, but when download starts it stuck at 0 kb/s

Also when I go task info I get this line:
https://i.imgur.com/LfsVizW.png

Same with technical preview

Also when I go save and quit I get this error now:
https://i.imgur.com/Fc5Pub6.png

**Comment by KurtBestor** [comment]
Are you using any custom plugins?

**Comment by 4Sol** [comment]
Yes, https://github.com/rickmiron/variado/blob/main/sankaku_downloader.py
+ kemono and something else from this guy
but when I laucnhed technical prewvie I just used sankaku plugin

Here is the task info on my last try without loging in (probably not admin launched):
https://pastebin.com/CNGuaw55

When I load cookies (or run it as an admin not sure which works) but it starts to count images, but speed is at 0 kb/s forever

It was working yesterday, not sure what's the matter

**Comment by rickmiron** [comment]
I have the same problem, but it is with this new version 4.1, the previous version does work.
I will check where the conflict may be, for now use the previous version 4.0

**Comment by KurtBestor** [comment]
Oh, I didn't take the case when the URL is `None` into account, I'll fix it in the next version.
If you modify `None` to `''` in the following part of the script, it should work in v4.1 as well:
https://github.com/rickmiron/variado/blob/d49efcde8a0b326026587fcb2bb024a2c6738b20/sankaku_downloader.py#L78

**Comment by rickmiron** [comment]
I also came to that conclusion, but I have not carried out many tests to verify that it will not cause problems since precisely that variable works as a 'referer' and in sankaku the files are downloaded more safely if they do not have a 'referer'

**Comment by rickmiron** [comment]
kurtBestor because when a png file is downloaded in self.single=True mode ffmpeg is activated , creates a jpg copy and then that copy is deleted, but it does not happen when it is jpg

**Comment by rickmiron** [comment]
4sol, Download the plugin again, it's already fixed

**Comment by 4Sol** [comment]
> 4sol, Download the plugin again, it's already fixed

I tried new version of the plugin.
It did count the images, but the download speed for me is still at 0 KB/s
Running as admin with login sankaku cookies

Maybe I'm blocked on sankaku or something? Don't know how to check it.

Pressing info still send me this error:
![image](https://github.com/KurtBestor/Hitomi-Downloader/assets/52103769/2a22d0e4-d45c-4cd9-89d5-450c943392f4)
And on exit from hitomi:
![image](https://github.com/KurtBestor/Hitomi-Downloader/assets/52103769/3236a3fe-bf95-4f3d-9786-c65a8edb9f2c)

**Comment by 4Sol** [comment]
A small update.

I tried again with both plugins and they are working now. Don't know what happened :D
1 sankaku_downloader 
2 sankaku_downloaderJ 

btw what is the difference between these 2 versions, plugin with "J" and without?
